website/addons/base/views.py

```diff
@@ -299,6 +303,8 @@ def create_waterbutler_log(payload, **kwargs):
 
         node_addon.create_waterbutler_log(auth, action, metadata)
 
+    file_signals.file_updated.send(user=user, node=node, event_type=action, payload=payload)
```

Comment:
The first positional argument to send is supposed to be the "sender". In this case, it probably makes the most sense to make `node` the sender.